x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/exceptions/exceptions_table.tsx

```diff
@@ -88,14 +124,45 @@ export const ExceptionListsTable = React.memo<ExceptionListsTableProps>(
       }) => async () => {
         try {
           setDeletingListIds((ids) => [...ids, id]);
+          if (refreshExceptions != null) {
+            await refreshExceptions();
+          }
+
+          if (exceptionsListsRef[id] != null && exceptionsListsRef[id].rules.length === 0) {
+            await deleteExceptionList({
+              id,
+              namespaceType,
+              onError: handleDeleteError,
+              onSuccess: handleDeleteSuccess,
+            });
+
+            if (refreshExceptions != null) {
+              refreshExceptions();
+            }
+          } else {
+            setReferenceModalState({
+              contentText: i18n.referenceErrorMessage(exceptionsListsRef[id].rules.length),
+              rulesReferences: exceptionsListsRef[id].rules.map(({ name }) => name),
+              isLoading: true,
+              listId: id,
```

Comment:
Looks like `listId` on line `118` above is an unused parameter. Could be cleaned up in [columns.ts](https://github.com/elastic/kibana/blob/3dfb1aba2a652b5f1464604e0e59302d9fa1d122/x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/exceptions/columns.tsx#L125) as well?

Reply:
Ah, yes, thank you! Cleaned up in both files.